tests/robot-tests/tests/admin/bau/prerelease_and_amend.robot

```diff
@@ -0,0 +1,302 @@
+*** Settings ***
+Library             Collections
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+Resource            ../../libs/admin/manage-content-common.robot
+Resource            ../../libs/tables-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev    AltersData
+
+
+*** Variables ***
+${TOPIC_NAME}=          %{TEST_TOPIC_NAME}
+${PUBLICATION_NAME}=    UI tests - prerelease and amend %{RUN_IDENTIFIER}
+${DATABLOCK_NAME}=      UI test table
+${RELEASE_NAME}=        Calendar Year 2000
+
+
+*** Test Cases ***
+Create test publication and release via API
+    ${PUBLICATION_ID}=    user creates test publication via api    ${PUBLICATION_NAME}
+    user creates test release via api    ${PUBLICATION_ID}    CY    2000
+
+Upload subject
```

Comment:
I think you can probably cut this part out (`Upload subject` and `Add metadata guidance`) as I don't think the subject is being used for the rest of the test?

We can get away with this because other Pre-release tests are doing the testing of Subjects more thoroughly.

This would also make this test run faster, which is always an added bonus :)

Reply:
I think it would be good to attach some data , to ensure reproduction of a realistic scenario for when the pre release viewer gets access to pre release content during amendment .

Reply:
My thoughts are that is tested more thoroughly in `prerelease.robot` and so isn't necessary to repeat here, but I could be wrong.  Worth checking?